TOC.md

```diff
@@ -122,6 +122,7 @@
     - [与 MySQL 的安全特性差异](dev/reference/security/compatibility.md)
     - [TiDB 数据库权限管理](dev/reference/security/privilege-system.md)
     - [TiDB 用户账户管理](dev/reference/security/user-account-management.md)
+    - [基于角色的访问控制](dev/reference/security/role-based-authentication.md)
```

Comment:
```suggestion
    - [基于角色的访问控制](dev/reference/security/role-based-authorization.md)
```
Please also update the filename to "role-based-authorization.md" because "authorization" means "授权" and "authentication" means "认证/鉴定".